# Vehicle dimensions

float32 length
float32 width
float32 turning_radius
float32 dist_rear_axle_numblade

